workflows/prognostic_c48_run/tests/test_validate.py

```diff
@@ -0,0 +1,72 @@
+from datetime import timedelta
+import cftime
+import pytest
+import runtime
+from runtime.validate import _validate_time_chunks, ConfigValidationError
+
+
+@pytest.fixture
+def initial_time():
+    return cftime.DatetimeJulian(2016, 8, 1)
+
+
+@pytest.fixture
+def timestep():
+    return timedelta(minutes=15)
+
+
+@pytest.fixture
+def run_duration():
+    return timedelta(hours=12)
```

Comment:
It seems unnecessary to use fixtures for these simple constants. Can you just hardcode in the function for now, and parameterize later if you need to?